gobcore/model/relations.py

```diff
@@ -338,3 +339,17 @@ def create_relation(src, validity, dst, derivation):
         f"dst{FIELD.ID}": dst["id"],
         f"dst_{FIELD.SEQNR}": dst[FIELD.SEQNR]
     }
+
+def get_catalog_collection_relation_name(model, rel_collection_name: str) -> Tuple[str, str, str]:
```

Comment:
nit: `tuple` will also do here, no need for import

Reply:
Ah yeah you're right. That's accepted these days ;) Changed it!